src/modules/deal-investor/deal-investor.service.ts

```diff
@@ -33,4 +35,33 @@ export class DealInvestorService {
       maximumLiability: investorInAcbs.LimitAmount,
     }));
   }
+
+  async createInvestorForDeal(dealIdentifier: string, newInvestor: CreateDealInvestorRequestItem): Promise<void> {
+    const idToken = await this.acbsAuthenticationService.getIdToken();
+
+    const investorToCreateInAcbs: AcbsCreateDealInvestorRequest = {
+      SectionIdentifier: PROPERTIES.DEAL_INVESTOR.DEFAULT.sectionIdentifier ?? '00',
+      EffectiveDate: this.dateStringTransformations.getEffectiveDate(newInvestor.effectiveDate),
+      ExpirationDate: this.dateStringTransformations.getExpirationDate(newInvestor.expiryDate),
+      IsExpirationDateMaximum: newInvestor.expiryDate ? false : true,
+      LenderType: {
+        LenderTypeCode: newInvestor.lenderType ?? PROPERTIES.DEAL_INVESTOR.DEFAULT.lenderType.lenderTypeCode ?? '100',
```

Comment:
Do we need to add hardcoded fallbacks as well as the constants (it was like this in the Mulesoft code)?

Reply:
No need for hardcoded values. 

Mulesoft code is a bit unusual in this mapping, maybe it was done so top layer (EA) API with constants could be skipped.

Reply:
In this line `LenderTypeCode: newInvestor.lenderType ?? PROPERTIES.DEAL_INVESTOR.DEFAULT.lenderType.lenderTypeCode ?? '100'` the hardcoded value (`100`) is not the same as the value in the properties file (`500`). I'm guessing the value in the properties file is 'right' and is the one we should use?

Reply:
In all other cases they are the same

Reply:
100 still looks like legacy Mulesoft thing. We can drop '100' and keep all default values in PROPERTIES

Reply:
Cool, I've dropped all the hardcoded values